src/system/game-data.ts

```diff
@@ -181,6 +183,16 @@ export const AbilityAttr = {
   ABILITY_HIDDEN: 4
 };
 
+export interface RunHistoryData {
+  [key: integer]: RunEntries;
+}
+
+export interface RunEntries {
+  entry: SessionSaveData;
+  victory: boolean;
+  favorite: boolean;
+}
```

Comment:
This should be singular. The plural indicates a list/array but this isn't one

```suggestion
export interface RunEntry {
  entry: SessionSaveData;
  victory: boolean;
  favorite: boolean;
}
```